test/v3/services/DeleteBFLossServiceSpec.scala

```diff
@@ -77,7 +77,7 @@ class DeleteBFLossServiceSpec extends ServiceSpec {
       "INVALID_IDVALUE"     -> NinoFormatError,
```

Comment:
This should be:
```suggestion
      "INVALID_TAXABLE_ENTITY_ID"     -> NinoFormatError,
```
plus whatever knock-on effects this has on service and controller code and specs.